### PR TITLE
Home and About page shuffle | CU-3rz12bj

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

Refactoring to allow space for project features and directly speaking to value offered and user painpoints solved.

**Notes for Reviewers**

This is the first of a few updates needed.

### CODE REVIEW

**Comment by l5io** [comment]
Task linked: [CU-3rz12bj Updates to Layer5.io](https://app.clickup.com/t/3rz12bj)

**Comment by l5io** [comment]
🚀 Preview for commit 874e2b6b56b877e180ada96d8ecec01a0198f895 at: https://62fe38bb9ddcfd2a5d9740c8--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 298a2f97f63a54becef5c2e0cc5d605a440b8a84 at: https://62fe405d7fe8f7317764d033--layer5.netlify.app

File: src/sections/Home/Layer5-statement/statement.style.js [hunk 1]
```diff
@@ -47,14 +52,13 @@ const BannerSectionWrapper = styled.section`
         color: rgba(255, 255, 255, 0.7);
     }
     .hero {
-        background: url(${imgHero});
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Layer5-statement/statement.style.js:50]: Now I get why we were seeing the JPG being downloaded twice in the networks stats earlier 😶

**Review by Nikhil-Ladha — COMMENTED** [review]
The MeshMap image in the `Layer5-Statement` has a thick white background (outline) which is causing the section to have white space on the right, can you edit the image to remove that and re-upload it?

File: src/pages/company/about.js [hunk 2]
```diff
@@ -25,7 +26,12 @@ const About = () => {
         <GlobalStyle />
         <SEO title="About" description='At Layer5, we believe collaboration enables innovation, and infrastructure enables collaboration. We help organizations look at their infrastructure differently, asking it "what have you done for me lately?"' />
         <Navigation theme={theme} themeSetter={themeSetter} />
+
         <AboutSection theme={theme} />
+
+
+        <CloudNativeLeaders />
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/pages/company/about.js:33]: Oh, good point.

File: src/sections/Home/Layer5-statement/statement.style.js (hunk 1 above)
Comment (leecalcote) [reply src/sections/Home/Layer5-statement/statement.style.js:50]: I had the same thought. :P I wonder what we were thinking at the time.

File: src/pages/company/about.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/pages/company/about.js]: Ok, I have moved the Stewards section back to the home page.

**Comment by l5io** [comment]
🚀 Preview for commit f0bccf71fb306a8029d9e7f08c94ccf2f0bad788 at: https://62fe864d8c8969005de64b22--layer5.netlify.app

**Comment by leecalcote** [comment]
> The MeshMap image in the Layer5-Statement has a thick white background (outline) which is causing the section to have white space on the right, can you edit the image to remove that and re-upload it?

I'm just noticing this comment. I'm looking now. Is this what you're referring to?

MeshmapDesigner copy 2

**Comment by Nikhil-Ladha** [comment]
> > The MeshMap image in the Layer5-Statement has a thick white background (outline) which is causing the section to have white space on the right, can you edit the image to remove that and re-upload it?
> 
> I'm just noticing this comment. I'm looking now. Is this what you're referring to?
> 
> MeshmapDesigner copy 2

Yes, though I just pushed a hack for it.
As I see the image has an inbuilt shadow which looks good in some places like on the MeshMap page. So, found a way to keep it and get the job done 😉

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by leecalcote** [comment]
The idea that your color blindness is subsiding has been sneaking up on me.

I have been bothered by these very same to erroneous white spaces as well. It makes me smile to see you point them out. Not only does it take someone proud of their work to point this out, but someone with a sharp eye for even the small pixels like these. 🔍

**Comment by Nikhil-Ladha** [comment]
Haha, what to say when I see the author as *Lee Calcote*, my eyes start rolling like anything 😄

**Pull request merged**